### DISCUSSION TITLE
Rembrandt 6800U mysterious APU power limits with iGPU load

### DISCUSSION THREAD

**Discussion by spariant** [FlyGoat/RyzenAdj#289] [General]
Minisforum EM680 (6800U+680M) seems to have a mysterious APU power limit when the iGPU is under load.
In CPU-only loads, the default 30W TDP applies. When iGPU is under load, this drops to 15W as reported by users. 
The OEM released an updated BIOS with a higher total system power profile (54W, borrowed from H series APUs), and this has moved the iGPU power up to 25W (20W with CPU boost disabled).

What is this strange power limit which handicaps the APU so severely when the iGPU is in use? 
It's not apu-slow-limit, which is set to 30 (info attached below). It's also not a memory/hardware bottleneck, as it was influenced by the BIOS update with system power settings adjustments. 

Attached are basic -info outputs with CPU-only and iGPU load. Any help identifying the right setting to raise CPU+iGPU performance will be greatly appreciated.

CPU-only load:
![ryzenadj_info_cpu](https://github.com/FlyGoat/RyzenAdj/assets/18563394/f8a86ca4-2348-4725-8721-df06422f0760)

iGPU load:
![ryzenadj_info_igpu](https://github.com/FlyGoat/RyzenAdj/assets/18563394/2ab08030-c43d-4d8c-ae27-35dedb0b959a)

**Comment by kimhuilim** [comment]
I noticed this situation as well with my HP laptop with Cezanne Zen 3. One thing that resolved it is changing STAPM value using Smokeless UMAF if your BIOS supports it.
